tests/test_array_api_utils.py

```diff
@@ -1,8 +1,8 @@
-import jax.numpy as jnp
 import numpy as np
 import pytest
 
 import glass._array_api_utils
+import jax.numpy as jnp
```

Comment:
I'm very confused what's going on with `isort` here

Reply:
`isort` thinks the `jax` import is coming from `glass.jax` (which is weird). This made me think if naming a submodule `jax` is wise, but then `jax` has a sub-package named `numpy` and they face no issues, so it should be okay.

I have explicitly added `jax` as a third-party import for `isort` and the imports look good now!

Reply:
Ah, yeah, that would be it. That is weird though, might be worth raising an issue on `isort`/`ruff`?